src/main/java/com/verapdf/restapi/service/JobService.java

```diff
@@ -0,0 +1,121 @@
+package com.verapdf.restapi.service;
+
+import ch.qos.logback.core.util.FileUtil;
+import com.verapdf.restapi.entity.Job;
+import com.verapdf.restapi.utils.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.UUID;
+
+
+@Service
+public class JobService {
+
+    private static final Logger log = LoggerFactory.getLogger(JobService.class);
+
+    private HashMap<UUID, Job> jobHashMap;
+
+    private ConfigService configService;
+
+    @Autowired
+    public JobService(ConfigService configService) {
+        jobHashMap = new HashMap<>();
+        this.configService = configService;
+    }
+
+    public void addJob(Job job) {
+        Path path = Paths.get(System.getProperty("user.dir"), configService.getSavePath(), File.separator, job.getJobId().toString());
+        FileUtils.createDirectory(path);
```

Comment:
1) as mensioned before: root job dir path shall be set in configuration file. 
2) during job creation we shall pass that dir into Job and create Job specific directory during Job creation inside that class.